frame/support/src/traits/hooks.rs

```diff
@@ -21,6 +21,25 @@ use impl_trait_for_tuples::impl_for_tuples;
 use sp_arithmetic::traits::Saturating;
 use sp_runtime::traits::AtLeast32BitUnsigned;
 
+/// Off-chain computation trait.
+///
+/// Implementing this trait on a module allows you to perform long-running tasks that make (by
+/// default) validators generate transactions that feed results of those long-running computations
+/// back on chain.
+///
+/// NOTE: This function runs off-chain, so it can access the block state, but cannot preform any
+/// alterations. More specifically alterations are not forbidden, but they are not persisted in any
+/// way after the worker has finished.
+#[impl_for_tuples(100)]
```

Comment:
make number of pallets in a runtime is 256 so we might as well make this 256

Reply:
Settled in #11813